graylog2-web-interface/src/views/components/searchbar/AbsoluteTimeRangeSelector.jsx

```diff
@@ -36,7 +36,7 @@ const _isValidDateString = (dateString: string) => {
   }
   return DateTime.isValidDateString(dateString)
     ? undefined
-    : `Invalid date: ${dateString}`;
+    : `Format must be: ${DateTime.Formats.TIMESTAMP}`;
```

Comment:
This is now `YYYY-MM-DD HH:mm:ss.SSS`. It should be `YYYY-MM-DD HH:mm:ss[.SSS]` to indicate that milliseconds are optional. As mentioned in a [previous comment](https://github.com/Graylog2/graylog2-server/pull/8371#discussion_r442686678), I think we cannot reuse an existing date format string, as we do combine multiple of them (two at the moment, more if we also make e.g. seconds optional).

Reply:
I thought the parentheses mark the difference, but I like the idea.
But when we mark a part as optional, shouldn't we display a message like `YYYY-MM-DD [HH:mm:ss[.SSS]]` since you can enter `2020-06-23` in the same way like `2020-06-23 10:10:37`?

Reply:
Yes, absolutely. I was not aware of being able to leave out the time, I only tested leaving out seconds, which unfortunately does not work yet :/